doc/examples.rst

```diff
@@ -0,0 +1,58 @@
+.. _pyXDSM_examples:
+
+Examples
+========
+Here is a simple example. 
+There are some other more advanced things you can do as well. 
+Check out the `examples folder <https://github.com/mdolab/pyXDSM/blob/master/examples>`_ for more complex scripts.
+
+.. code-block:: python
```

Comment:
Is this the same as one of the examples? If so, you should import the code here instead of copy/paste, with something like
```
.. literalinclude:: <path-to-file>
```

Reply:
Good catch, I fixed a couple of those from the readme but I missed this